README.md

```diff
@@ -140,6 +140,31 @@ quark -a first.apk -a second.apk -C
 
 <img src="https://i.imgur.com/ClRWOei.png"/>
 
+### Radiocontrast
+The Radiocontrast is a Quark API that can quickly generate 100% confidence rules with the Native APIs used in a specific method. The feature lets you easily see through the behavior of a method, just like radiocontrast.
+
+For example, if you want to see the behavior method in `Ahmyth` called `Lahmyth/mine/king/ahmyth/CameraManager;->startUp(I)V,` 
```

Comment:
+ There seems to be a missing word between "behavior" and "method".
+ I suggest exchanging the part, "in Ahmyth", and the part, "called Lahmyth/mine/king/ahmyth/CameraManager;->startUp(I)V," to make the sentence clear, like the following.
```
For example, we want to know the behavior of a method called `Lahmyth/mine/king/ahmyth/CameraManager;->startUp(I)V,` in Ahmyth.apk.
```